components/automate-grpc/scripts/bootstrap_protoc.sh

```diff
@@ -13,10 +13,10 @@ set -e
 
 
 printf 'GEN: %s\n' components/automate-grpc/protoc-gen-a2-config/api/a2conf/*.proto
-protoc -I /src --go_out=logtostderr=true,paths=source_relative:/src \
```

Comment:
`logtostderr` went away as an option. I think it was in this commit: https://github.com/golang/protobuf/commit/aaf9c3b54ff2d3e40f85540b0c98b0d2f5fbaf34#diff-8c603013608023320d5242916c4ea03b

It's tricky to follow because they pointed the CLI at a new implementation that didn't support all the same flags instead of modifying the existing one. In any case, if you look at the godoc for the new implementation (e.g., https://godoc.org/google.golang.org/protobuf/compiler/protogen#Options.Run) all the functions say they log errors and whatnot to stderr. So tl;dr the thing we are asking for is now the default behavior and the flag to enable it is gone.